src/config/quantTypes.json

```diff
@@ -384,6 +384,58 @@
       ]
     }
   },
+  "library10XVDJ": {
+    "key": "library10XVDJ",
+    "name": "Library - 10X VDJ - Tapestation",
+    "wellType": "TS10XVDJ",
+    "parse": {
+      "delimiter": ",",
+      "rowDelimiter": ["\r\n", "\r", "\n", "\r\r\n"],
+      "from": 2,
+      "columns": [
+        "filename",
+        "wellId",
+        "sampleDescription",
+        "fromBasePair",
+        "toBasePair",
+        "averageSizeBasePairs",
+        "concentration",
+        "regionMolarity",
+        "percentageOfTotal",
+        "regionComment"
+      ]
+    },
+    "metadata": {},
+    "conversion": {
+      "factors": {
+        "dilution": 10
+      },
+      "expression": "(dilution)",
+      "decimalPlaces": 3
+    },
+    "qcResults": {
+      "key": "molarity",
+      "units": "nM",
+      "assay": {
+        "type": "Library - X10 VDJ",
+        "version": "v1.0"
+      },
+      "outlier": {
+        "type": "cv",
+        "threshold": 15
```

Comment:
This gives `0.0` for all the samples.  I don't know if it's the right type of outlier detection to use or if we need an outlier section at all.  Some other quantTypes don't have one.

Reply:
Would 0.0 detect any outliers? I'm sorry again I don't know enough about Quanthub!

Reply:
No it doesn't.  Maybe I should be clearer "I'm certain this isn't the right type of outlier detection, but I don't know whether we want to remove outlier detection completely for these or to use something different."  I'm not sure whether these plates contain repetitions, but if they don't, then I suspect there's nothing to base outlier detection on.

Reply:
Also confirm with user. Is source 96-well plate stamped with replicates into 384-well plate? What 384-well layout is that, how many replicates are there and where etc. Current outlier detection is about spotting outliers in replicates of 3. May not be the same here.
Threshold value if needed is also down to user to determine.